src/main.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
+from config import DEFAULT_LLM_MODEL, CSV_FILE_PATH
+from constants import (
+    OPTIONS,
+    INSTRUCTION,
+    PROMPT,
+    INPUT_TEXT_1,
+    INPUT_TEXT_2,
+    INPUT_TEXT_3,
+    INPUT_TEXT_4,
+    INPUT_TEXT_5,
+    INPUT_TEXT_6,
+    INPUT_TEXT_7,
+    INPUT_TEXT_8,
+    INPUT_TEXT_9,
```

Comment:
what gonna happen if user want to add 5 more validation examples? I guess he will need to go and change the code. It will be time consuming. Number of inputs should be read dynamicly from CSV.